### PR TITLE
RIA-3718: Add remote hearing option to Submit hearing requirement

### PR DESCRIPTION
**Pull request by divisathyan**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3718

### Change description ###
LR adds remote hearing option to Submit hearing requirement

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: e2e/features/submit-hearing-requirements.feature [hunk 1]
```diff
@@ -32,7 +32,6 @@ Feature: Submit hearing requirements
     And I should see the text `The appeal is now going to a hearing. You need to submit the appellant's hearing requirements.`
     And I should see the case details
     And I should not see the hearing details
-    And I should see the legal representative details
```
Comment (bradsorour) [inline e2e/features/submit-hearing-requirements.feature:35]: Is there any reason we do not want to see these details anymore?

File: e2e/features/submit-hearing-requirements.feature [hunk 2]
```diff
@@ -48,7 +47,6 @@ Feature: Submit hearing requirements
 
     When I click the `Appellant` tab
     Then I should see the appellant's details
-    And I should see the legal representative's details
```
Comment (bradsorour) [inline e2e/features/submit-hearing-requirements.feature:51]: Same as above?

**Review by bradsorour — COMMENTED** [review]
Please confirm if we no longer need to see legal rep details on the `Overview` and `Appellant` tabs.

File: e2e/features/submit-hearing-requirements.feature (hunk 1 above)
Comment (Lee365) [reply e2e/features/submit-hearing-requirements.feature:35]: Added - should see the legal representative details

File: e2e/features/submit-hearing-requirements.feature (hunk 2 above)
Comment (Lee365) [reply e2e/features/submit-hearing-requirements.feature:51]: Added - should see the legal representative details

**Review by bradsorour — APPROVED** [review]
LGTM

**Review by mkayad — APPROVED** [review]
LGTM

**Pull request merged**